bootstrap/main.go

```diff
@@ -60,7 +60,7 @@ func mainImpl() {
 	}
 
 	if !kubectlClient.IsPresent() {
-		exitNoCapture("Could not find kubectl in PATH, please install it: https://kubernetes.io/docs/tasks/tools/install-kubectl/\n")
+		exitWithCapture("Could not find kubectl in PATH, please install it: https://kubernetes.io/docs/tasks/tools/install-kubectl/\n")
```

Comment:
This one should be safe to ignore, no?

Reply:
That's what I thought before in the other case, but that assumes that kubectlClient.IsPresent()` is working correctly.
It's probably better to set rate-based ignore filter in sentry, than rely on our judgement.

Reply:
We're just looking in the path in `IsPresent`: https://github.com/weaveworks/launcher/blob/master/pkg/kubectl/local.go#L15. Can't quite think of a case where we want to know about a failure here.

Reply:
My point is that we should not do a case-by-case evaluation of whether we think something should be captured, because sometimes we'll get it wrong.
It's probably less error-prone to ignore it in sentry, and it gives us visibility of how often something happens, which might be usefull for targeted UX improvements etc.